test/e2e/webhook_test.go

```diff
@@ -424,3 +426,738 @@ var _ = Describe("webhook tests for MC taints", Ordered, func() {
 		}, testutils.PollTimeout, testutils.PollInterval).Should(Succeed())
 	})
 })
+
+var _ = Describe("webhook tests for ClusterResourceOverride", Ordered, func() {
+	Context("Test CRO CREATE operations", Ordered, func() {
+		croName := "cro-test"
```

Comment:
we need to use name similar to crp format, fmt.Sprintf(crpNameTemplate, GinkgoParallelProcess())

otherwise, when tests run in parallel. They will impact each other.